tests/api/test_states.py

```diff
@@ -0,0 +1,92 @@
+import unittest
+import json
+
+from app import app, db
+from models import Category, Criterion, Link, Score
+from datetime import datetime, timedelta
+from tests.test_utils import clear_database, create_category, create_criterion, create_link
+
+
+class StatesTestCase(unittest.TestCase):
+    def setUp(self):
+        self.client = app.test_client()
+
+        self.category1 = create_category()
+        self.category2 = create_category()
+        self.criterion1 = create_criterion(self.category1.id)
+        self.criterion2 = create_criterion(self.category2.id)
+        self.link1 = create_link(self.category1.id, 'NY')
+        self.link2 = create_link(self.category2.id, 'NY')
+        self.score1 = Score(
+          criterion_id=self.criterion1.id,
+          state='NY',
+          meets_criterion=True
+        )
+        self.score2 = Score(
+          criterion_id=self.criterion2.id,
+          state='NY',
+          meets_criterion=False
+        )
+        self.score3 = Score(
+          criterion_id=self.criterion2.id,
+          state='NY',
+          meets_criterion=True
+        )
+        # score2 is more recent than score3
+        self.score3.created_at = datetime.utcnow() - timedelta(5)
+        self.score4 = Score(
+          criterion_id=self.criterion2.id,
+          state='AK',
+          meets_criterion=True
+        )
+
+        Category.save_all([self.category1, self.category2])
+        Criterion.save_all([self.criterion1, self.criterion2])
+        Link.save_all([self.link1, self.link2])
```

Comment:
Since objects are now saved in the `create_` utility functions, we don't need this bit!